tinkergraph-gremlin/src/main/java/org/apache/tinkerpop/gremlin/tinkergraph/structure/TinkerGraph.java

```diff
@@ -103,9 +103,9 @@ public final class TinkerGraph implements Graph {
     protected TinkerIndex<TinkerVertex> vertexIndex = null;
     protected TinkerIndex<TinkerEdge> edgeIndex = null;
 
-    protected final IdManager<?> vertexIdManager;
-    protected final IdManager<?> edgeIdManager;
-    protected final IdManager<?> vertexPropertyIdManager;
+    protected IdManager<?> vertexIdManager;
+    protected IdManager<?> edgeIdManager;
+    protected IdManager<?> vertexPropertyIdManager;
```

Comment:
Is the removal of `final` here required?

Reply:
I tried to explain this in the commit message.

> The final modifier was removed in some TinkerGraph manager fields as these were removed during testing with reflection. However, as of Java 18 there is no reasonable way to use reflection to do this. Also, the class is marked final which should reduce its impact.

Please let me know if you have further concerns about this.